transform/mattermost-analytics/models/staging/mm_telemetry_rc/stg_mm_telemetry_rc__performance_events.sql

```diff
@@ -1,15 +1,9 @@
 {%- set include_columns = ["context_page_referrer", "channel", "context_app_namespace", "user_actual_id"
 , "context_library_name", "type", "context_app_version" , "user_actual_role", "context_page_url", "context_os_name" 
```

Comment:
The following columns can also be removed: `context_page_path`, `context_os_name`, `context_os_version`, `context_page_referrer`, `context_page_search`, `context_page_title`, `context_page_url`.

They contain < 10 non null values and they are all junk.

Reply:
Removed these